### PR TITLE
Small edits when going through gsoc 2022 notebooks

### PR DESCRIPTION
**Pull request by leewujung**
This PR includes small changes and notes as I go through notebooks created by @ldr426 as part of his gsoc2022 work.

This is WIP as I continue to push up commits and comments.

### CODE REVIEW

**Comment by ldr426** [comment]
merge before new changes

**Review by ldr426 — COMMENTED** [review]
Reviewed

**Review by valentina-s — APPROVED** [review]

File: docs/source/gsoc/2022/Echogram_EchoStats.ipynb [hunk 1]
```diff
@@ -2180,21 +2181,22 @@
...
+    "#             print(f\"Failed to process raw file {raw_fpath.name}: {e}.\")\n",
+    "#             break\n",
     "        \n",
     "        try:\n",
     "            # Access file directly from S3 to create a converted EchoData object in memory\n",
-    "            ed = ep.open_raw(\n",
-    "                f\"{desired_raw_file_url}\",\n",
-    "                sonar_model='ek60'\n",
-    "            )\n",
+    "            ed = ep.open_raw(f\"{desired_raw_file_url}\", sonar_model='ek60')\n",
```
Comment (valentina-s) [inline docs/source/gsoc/2022/Echogram_EchoStats.ipynb:2190]: @
```suggestion
                 # opening something
    "            ed = ep.open_raw(f\"{desired_raw_file_url}\", sonar_model='ek60')\n",
```

Comment (leewujung) [reply docs/source/gsoc/2022/Echogram_EchoStats.ipynb:2190]: It is something

**Pull request merged**